.github/workflows/check_doxygen_awesome_version.yml

```diff
@@ -21,7 +23,7 @@ jobs:
           # NOTE: In a later runs of CI we will search for PR with this exact
           #       title. Only if no such PR exists will the script create a
           #       new PR.
-          PR_TITLE="Update doxygen-awesome to ${VERSION}"
+          PR_TITLE="[docs] Update doxygen-awesome to ${VERSION}"
```

Comment:
This doesn't seem to cause issues in the test repo. Hence, it should be fine. It will also not generate a new PR for the current version since there's no difference it doesn't try to create a PR.